GitUI/Avatars/GravatarProvider.cs

```diff
@@ -14,6 +14,14 @@ public sealed class GravatarProvider : IAvatarProvider
         private readonly AvatarFallbackType? _fallback;
         private readonly bool _forceFallback;
 
+        #pragma warning disable SYSLIB0021 // 'MD5CryptoServiceProvider' is obsolete
+
+        // Hash specified at http://en.gravatar.com/site/implement/hash/
+        private readonly MD5CryptoServiceProvider _md5 = new();
```

Comment:
The warning message gives a hint about easy-to-use replacement.

Reply:
Thanks for the head up. I just moved the code without a careful look and was thinking that it was the MD5 hashing algorithm that was obsoleted. In fact, it was just the API. I will push the fix...

Reply:
done.